bundle/compliance/cis_aws/rules/cis_3_5/test.rego

```diff
@@ -0,0 +1,36 @@
+package compliance.cis_aws.rules.cis_3_5
+
+import data.cis_aws.test_data
+import data.compliance.cis_aws.data_adapter
+import data.compliance.policy.aws_config.ensure_config_enabled as audit
+import data.lib.test
+
+finding = audit.finding
+
+test_violation {
+	eval_fail with input as rule_input(false, false)
+	eval_fail with input as rule_input(true, false)
+	eval_fail with input as rule_input(false, true)
+}
+
+test_pass {
+	eval_pass with input as rule_input(true, true)
+}
+
+test_not_evaluated {
+	not_eval with input as test_data.not_evaluated_trail
+}
```

Comment:
should we rename `not_evaluated_trail` to smth like `unknown_object_type` so all `test_not_evaluated` cases can use it to verify the rule only operates on a specific type/subType? 

didn't want to create a dummy object just for my case, so reused this one but the name is a bit misleading.